README.md

```diff
@@ -40,7 +40,7 @@ If you want to contribute to this list (please do), send me a pull request.
 * [react-worker-dom](https://github.com/web-perf/react-worker-dom) - Experiments to see the advantages of using Web Workers to Render React Virtual DOM.
 * [rax](https://github.com/alibaba/rax) - A universal React-compatible render engine.
 * [react-native-dom](https://github.com/vincentriemer/react-native-dom) An experimental, comprehensive port of React Native to the web.
-
+* [react-regl](https://kevzettler.com/react-regl) Seamlessly Render raw 2D or 3D WebGL shaders to a React DOM.
```

Comment:
@kevzettler please provide a GitHub link here.

Reply:
whoops updated in https://github.com/chentsulin/awesome-react-renderer/pull/45/commits/631dab3983bb616661ecea64f0bbbd5227c5a2d7 @chentsulin